app/assets/stylesheets/themes/law.scss

```diff
@@ -36,9 +36,9 @@
 
   @include bp(m) {
     .ie8 .subNavLinks {
-      border-right: 1px solid #00307c;
-      border-bottom: 1px solid #00307c;
-      border-left: 1px solid #00307c;
+      border-right: 1px solid $cu-light-grey;
```

Comment:
This file was a hotfix for Ross, who noticed blue hover colors on https://www.chapman.edu/law/student-resources/library/interlibrary-loan.aspx